test.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
```

Comment:
the main index.html should more or less mirror this structure. 

The button are a TODO ( for instance we could trigger a rerun of failing test only.